specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,39 +15,36 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+Create an open source frontend for the VDK.
```

Comment:
Please split out the VEP in a separate PR. 

We want to break code commits into small self-contained units which can be merged quickly and independently.